runtime/admin/src/main/resources/application.properties

```diff
@@ -49,9 +56,10 @@ quarkus.index-dependency.guava.group-id=com.google.guava
 quarkus.index-dependency.guava.artifact-id=guava
 quarkus.index-dependency.protobuf.group-id=com.google.protobuf
 quarkus.index-dependency.protobuf.artifact-id=protobuf-java
-quarkus.datasource.devservices.image-name=postgres:17-alpine
 
 ## Logging Configuration
 # Set root logging level to WARN to avoid mostly irrelevant Quarkus log messages.
 # The Admin tool communicates with the user via STDOUT/STDERR
-quarkus.log.level=WARN
\ No newline at end of file
+quarkus.log.level=WARN
```

Comment:
Looks like the test-port settings aren't necessary - seems there was something else at play